src/main/java/com/google/devtools/build/lib/runtime/ExperimentalStateTracker.java

```diff
@@ -206,8 +210,8 @@ void buildComplete(BuildCompleteEvent event, String additionalInfo) {
       } else {
         additionalMessage =
             additionalInfo + "Build completed, "
-            + failedTests + " test" + (failedTests == 1 ? "" : "s") + " FAILED, "
-            + actionsCompleted + " total action" + (actionsCompleted == 1 ? "" : "s");
+                + failedTests + " test" + (failedTests == 1 ? "" : "s") + " FAILED, "
```

Comment:
Revert the layout changes in this file?

Reply:
file changes reverted - since they are no longer relevant